styleguide/index.erb

```diff
@@ -632,6 +587,51 @@
       </li>
     </ul>
     <% end %>
+
+    <% styleguide_block  'Gallery - Mosaic' do %>
+    <ul class="gallery -quartet -mosaic">
+        <li>
```

Comment:
Definitely not a big deal but the indentation looks like it is 4 spaces instead of 2? It seems like we have been using two spaces everywhere else.